### PR TITLE
Removing Tester Flag / Deploying Header to everyone.

### PR DESCRIPTION
**Pull request by ovbm**
Removed tester flag and replaced old with new components / fixed a mi…nor header icon padding bug

### CODE REVIEW

**Comment by ovbm** [comment]
Screenshot 2020-07-16 at 14 34 41

File: components/Article/Page.js [hunk 1]
```diff
@@ -736,10 +723,8 @@ class ArticlePage extends Component {
           meta && meta.discussionId && router.query.focus ? undefined : meta
         }
         onNavExpanded={this.onPrimaryNavExpandedChange}
-        secondaryNav={seriesNavButton || (!isTester && actionBarNav)}
-        showSecondary={
-          isTester && seriesNavButton ? true : this.state.showSecondary
-        }
+        secondaryNav={seriesNavButton}
+        showSecondary={seriesNavButton ? true : this.state.showSecondary}
```
Comment (tpreusse) [inline components/Article/Page.js:727]: Is `this.state.showSecondary` still necessary? IMO can be removed (also from initial state)

File: components/Article/Page.js [hunk 2]
```diff
@@ -640,10 +627,10 @@ class ArticlePage extends Component {
       article.content.meta &&
       article.content.meta.darkMode
 
-    const MySeriesNavButton = isTester ? SeriesNavButtonNew : SeriesNavButton
+    console.log(meta)
```
Comment (tpreusse) [inline components/Article/Page.js:630]: rm

File: components/Frame/Popover/UserNav.js [hunk 3]
```diff
@@ -73,10 +72,10 @@ const UserNav = ({
                   </div>
                 </>
               )}
-              {!isMember && (
+              {!isMember && !inNativeApp && (
                 <Button
```
Comment (tpreusse) [inline components/Frame/Popover/UserNav.js:76]: Use `Link` here for fast client-side-navigation, re add the import from `lib/routes` and then do:

```
<Link route='pledge' passHref>
  <Button {...}>{...}</Button>
</Link>
```

File: components/Frame/Popover/Nav.js [hunk 4]
```diff
@@ -1,320 +1,186 @@
...
+          <>
+            {!me && (
+              <>
+                <div {...styles.signInBlock}>
+                  <SignIn style={{ padding: 0 }} />
+                </div>
+              </>
+            )}
+            {!isMember && !inNativeApp && (
+              <Button style={{ marginTop: 24 }} href='/pledge' black block>
```
Comment (tpreusse) [inline components/Frame/Popover/Nav.js:46]: also use `<Link>` here like UserNav comment

**Comment by ovbm** [comment]
Screenshot 2020-07-16 at 17 33 27
Screenshot 2020-07-16 at 17 33 18

**Pull request merged**